dev-libs/tinyxml2/tinyxml2-6.0.0.recipe

```diff
@@ -0,0 +1,69 @@
+SUMMARY="A small C++ XML Parser"
+DESCRIPTION="Tinyxml is a simple, OS independent XML parser for the C++ \
+language.
+Timyxml was first created to solve the common text I/O file problem. It \
+parses the XML into a DOM-like tree and is able to read and write XML files.\
+It allows you to create your own document mark-ups or even construct an XML \
+document from scratch with C++ objects."
+HOMEPAGE="http://www.grinninglizard.com/tinyxml2/"
+COPYRIGHT="2011-2017 Lee Thomason"
+LICENSE="Zlib"
+REVISION="1"
+SOURCE_URI="https://github.com/leethomason/tinyxml2/archive/$portVersion.tar.gz"
+CHECKSUM_SHA256="9444ba6322267110b4aca61cbe37d5dcab040344b5c97d0b36c119aa61319b0f"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	tinyxml2$secondaryArchSuffix = $portVersion
```

Comment:
Is it API/ABI compatible with 2.x

Reply:
@extrowerk can you recheck this one and if ok get it merged?

Reply:
There was 4 soname change since version 2.2, so al lthe dependant packages needs to be tested.
According grep only 3 package depends on it: fife, blender, opencolorio
@nyssss : can you test them?
In the same time you could bump it to 6.2.0.